

This is the genome-scale metabolic network of Plasmodium falciparum described
in the article:  
**Reconstruction and flux-balance analysis of the Plasmodium falciparum metabolic network.**   
Plata G, Hsiao TL, Olszewski KL, Llinás M, Vitkup D.; Mol Syst Biol. 2010 Sep
7;6:408.; PmID: [20823846](http://www.ncbi.nlm.nih.gov/pubmed/20823846) , DOI:
[10.1038/msb.2010.60](dx.doi.org/10.1038/msb.2010.60)  
Abstract:  
Genome-scale metabolic reconstructions can serve as important tools for
hypothesis generation and high-throughput data integration. Here, we present a
metabolic network reconstruction and flux-balance analysis (FBA) of Plasmodium
falciparum, the primary agent of malaria. The compartmentalized metabolic
network accounts for 1001 reactions and 616 metabolites. Enzyme-gene
associations were established for 366 genes and 75% of all enzymatic
reactions. Compared with other microbes, the P. falciparum metabolic network
contains a relatively high number of essential genes, suggesting little
redundancy of the parasite metabolism. The model was able to reproduce
phenotypes of experimental gene knockout and drug inhibition assays with up to
90% accuracy. Moreover, using constraints based on gene-expression data, the
model was able to predict the direction of concentration changes for external
metabolites with 70% accuracy. Using FBA of the reconstructed network, we
identified 40 enzymatic drug targets (i.e. in silico essential genes), with no
or very low sequence identity to human proteins. To demonstrate that the model
can be used to make clinically relevant predictions, we experimentally tested
one of the identified drug targets, nicotinate mononucleotide
adenylyltransferase, using a recently discovered small-molecule inhibitor.

To make this model SBML compliant, the unit for the FLUX_VALUE parameters had
to be changed from mmole per gramm dry weight per hour to mmole per hr. To
change the model back to its original form, either alter the unit definition
of the unit mmol_per_hr accordingly or replace mmol_per_hr by
mmol_per_gDW_per_hr in all reactions.

This model originates from BioModels Database: A Database of Annotated
Published Models (http://www.ebi.ac.uk/biomodels/). It is copyright (c)
2005-2011 The BioModels.net Team.  
To the extent possible under law, all copyright and related or neighbouring
rights to this encoded model have been dedicated to the public domain
worldwide. Please refer to [CC0 Public Domain
Dedication](http://creativecommons.org/publicdomain/zero/1.0/) for more
information.

In summary, you are entitled to use this encoded model in absolutely any
manner you deem suitable, verbatim, or with modification, alone or embedded it
in a larger context, redistribute it, commercially or not, in a restricted way
or not..  
  
To cite BioModels Database, please use: [Li C, Donizelli M, Rodriguez N,
Dharuri H, Endler L, Chelliah V, Li L, He E, Henry A, Stefan MI, Snoep JL,
Hucka M, Le Novère N, Laibe C (2010) BioModels Database: An enhanced, curated
and annotated resource for published quantitative kinetic models. BMC Syst
Biol., 4:92.](http://www.ncbi.nlm.nih.gov/pubmed/20587024)

